release_notes/45-release-notes.adoc

```diff
@@ -376,13 +377,31 @@ The following section provides information about removed features listed in the
 * Direct upgrades from {product-title-short} version 3.74 or earlier to version 4.5 are no longer supported. To upgrade to version 4.5 or later, you must first upgrade to version 4.4.
 
 [id="bug-fixes_{context}"]
-== Bug fixes
-
-[id="resolved-in-version-450_{context}"]
-=== Resolved in version 4.5.0
+== Bug fixes in version 4.5.0
 
 *Release date*: 24 July 2024
 
 * Previously, a bug prevented users without administrator privileges from accessing the listening endpoint data for clusters and namespaces for which they had `read` permission. With this update, users with `read` permission can now receive data from the listening endpoints service.
 
+[id="about-release-451_{context}"]
+== About release 4.5.1
+
+This release includes notable technical changes and bug fixes.
+
+[id="changes-in-version-451_{context}"]
+=== Changes in version 4.5.1
+
+* The network graph and the network policy generator in the {product-title-short} portal were updated to clarify that `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources are not taken into consideration in the network graph or during network policy generation.
+
+* The field `error` returned for failed API calls has been deprecated, and will be removed in a future release. Instead of using the `error` field, use the `message` field. The `message` field contains the same information as the `error` field.
```

Comment:
```suggestion
* The field `error` returned for failed API calls has been deprecated, and is planned for removal in a future release. Instead of using the `error` field, use the `message` field. The `message` field contains the same information as the `error` field.
```
[Per the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#deprecated-and-removed-features), we should say "plan to remove" rather than a definitive "will be removed" in our docs.